content/departments/engineering/design/goals/index.md

```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team for the remainder of Q4.
+
+Our team-level goals cascade from the overall company [product vision and
+strategy](../../index.md#product-vision-and-strategy), and each member of the design team
+uses these to define their own individual goals.
+
+## (Draft) Design team goals for Q4
+
+### Urgent and important
+
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the strategic scale
+     audit.
+   - Key Result: Design as a function contributes to closing a deal for a
+     2XL customer.
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least 14 hours of
```

Comment:
Note, I set the time goal to 14 hours of user-facing time counting 7 weeks until Jan 31. I've discounted 2 weeks due to Thanksgiving and Christmas.